### ISSUE TITLE
Migrate D7 - MenuLinks options are not being migrated by (patched) D2D migrate module

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#343]
Noticed that menu options are not being migrated by DrupalMenuLinks7Migration class.

This may be the reason we are seeing issues with certain menus not showing up (eg. Economics People menu, global menu items on the Arts department sites, etc.)

After running drush mm UGMenuLinks7, options is listed under the DNM (Do Not Migrate) section. After further inspection, it looks like the person who put together the patch which made the original DrupalMenuLinks7Migration class work (see under D2D migrate folder under d7/menu_links) added the following comment: "Ran into trouble with options, so I'm leaving it out for now", after which he or she added the options to the DNM section.

Investigate whether we should migrate these options. Technically we can by mapping the options field, however, given the comment, we should determine if there are significant issues if we do.

#Workaround
Fix any menu issues caused by the lack of options manually.

**Comment by mmafe** [comment]
See https://www.drupal.org/node/1489858 for more info on the patch.

**Comment by mmafe** [comment]
Possible example: History AODA vs History legacy menu links.

Faculty & Staff is not showing up in the secondary-links. The only difference between History AODA and History Legacy for that menu link is that the options have not been brought over.

**Comment by mmafe** [comment]
**Update:** Menu link options do not seem to be the cause of the issue where system menu links were not showing up in the menu. It turned out the system menu links *were* being brought over, but we just needed to create a corresponding view or page (as had been done on the source site) that connected up to the system menu link.

The main reason this issue was opened was because the system menu links were not displaying correctly. Since that problem has been solved, I'm going to consider the lack of menu options to be a *won't fix" for the migration module (particularly since the patch creator specifically chose *not* to bring over the menu options due to the trouble it caused).
